internal/k8sdeps/configmapandsecret/kv.go

```diff
@@ -100,3 +100,12 @@ func kvFromLine(line []byte, currentLine int) (kvPair, error) {
 	kv.key = key
 	return kv, nil
 }
+
+func stripQuotes(s string) string {
```

Comment:
You can use `strings.Trim`, similar to this line:
https://github.com/kubernetes-sigs/kustomize/blob/111f41785f73661e33ca81f0020fe3a4b5f00e92/internal/k8sdeps/configmapandsecret/configmapfactory.go#L185

Reply:
Thought about this but do we want to trim just one level of quotes or leading and trailing? e.g. FOO="""BAR"""